examples/44_cloudflare/src/entries.tsx

```diff
@@ -1,56 +1,159 @@
-import { unstable_defineEntries as defineEntries } from 'waku/minimal/server';
```

Comment:
I'm not sure what's the original intention of using waku/minimal/server over createPages, but maybe it's fine. cc @rmarscher

Reply:
I think I was just copying other examples that seemed to limit the surface area of the test by using the minimal API.

I was actually working on a new example that could be used a template for Cloudflare projects when using create-waku or c3. It's similar to the default template but with a few extra Cloudflare-specific examples and the dev server middleware.